packages/spectral/src/util.ts

```diff
@@ -365,6 +370,14 @@ export const lowerCaseHeaders = (
     return { ...result, [key.toLowerCase()]: val };
   }, {});
 
+/** This function accepts an arbitrary object/value and safely serializes it.
+ *
+ * @param value Arbitrary object/value to serialize.
+ * @returns JSON serialized text that can be safely logged.
+ */
+export const serializeObject = (value: unknown): string =>
+  JSON.stringify(serializeError(value), null, 2);
+
```

Comment:
I am confused about the purpose of this function. Its name does not seem to match up with its implementation.

Reply:
Definitely open to suggestions - the intent here is to provide utilities to aid in handling arbitrary payloads, particularly for serialization use cases. We've seen several usages of `JSON.parse(JSON.stringify(...))` in an effort to "strip out" unserializable values (functions, cyclic references, and so on). `serializeError` handles the cycles and a few other use cases though it does have some behaviors that are likely undesirable for this use case. 

I'll look at finding a more suitable replacement.